provider/blockdev_commit_base.py

```diff
@@ -0,0 +1,153 @@
+import logging
```

Comment:
Put BlockDevCommitTest into provider, then I guess more test cases besides blockdev_commit.py will employ BlockDevCommitTest to run tests in future, right?

Reply:
Yes, you're right.